go-controller/pkg/ovn/egressgw.go

```diff
@@ -681,24 +681,24 @@ func addOrUpdatePodSNATOps(nbClient libovsdbclient.Client, nodeName string, extI
 // by ecmp routes.
 // WARNING: updates same db entries as apbroutecontroller. Make sure to call only when route is not managed by
 // apbroute controller.
-func (oc *DefaultNetworkController) addHybridRoutePolicyForPod(podIP net.IP, node string) error {
+func (oc *DefaultNetworkController) addHybridRoutePolicyForPod(podIP, node string) error {
```

Comment:
Same. keep the `net.IP` until the very last moment?

Reply:
done.